lib/recognizer_web/controllers/accounts/user_settings_controller.ex

```diff
@@ -31,17 +32,55 @@ defmodule RecognizerWeb.Accounts.UserSettingsController do
     end
   end
 
+  def resend(conn, _params) do
+    current_user = Authentication.fetch_current_user(conn)
+    current_time = System.system_time(:second)
+
+    conn = put_session(conn, :two_factor_issue_time, current_time)
+
+    conn
+    |> send_two_factor_notification(current_user)
+
+    conn
+    |> put_flash(:info, "Two factor code has been resent")
+    |> redirect(to: Routes.user_settings_path(conn, :two_factor_confirm))
+
+    # |> redirect(external: Application.get_env(:recognizer, :redirect_url))
```

Comment:
nitpick: in the future, please delete, rather than comment out. it helps with the diff.